plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -66,12 +90,19 @@ def stub_add_changeset
+  let(:jenkins_xml_with_string_build_params_other_then_samson) { "<?xml version='1.0' encoding='UTF-8'?>\n<project>\n  <actions/>\n  <description></description>\n  <keepDependencies>false</keepDependencies>\n  <properties>\n    <com.sonyericsson.rebuild.RebuildSettings plugin=\"rebuild@1.25\">\n      <autoRebuild>false</autoRebuild>\n      <rebuildDisabled>false</rebuildDisabled>\n    </com.sonyericsson.rebuild.RebuildSettings>\n    <hudson.model.ParametersDefinitionProperty>\n      <parameterDefinitions>\n        <hudson.model.StringParameterDefinition>\n          <name>test1</name>\n          <description>this is sample desc</description>\n          <defaultValue>default value</defaultValue>\n        </hudson.model.StringParameterDefinition>\n        <hudson.model.StringParameterDefinition>\n          <name>test2</name>\n          <description>default value 3</description>\n          <defaultValue>default value 2</defaultValue>\n        </hudson.model.StringParameterDefinition>\n      </parameterDefinitions>\n    </hudson.model.ParametersDefinitionProperty>\n  </properties>\n  <scm class=\"hudson.scm.NullSCM\"/>\n  <canRoam>true</canRoam>\n  <disabled>false</disabled>\n  <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>\n  <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>\n  <triggers/>\n  <concurrentBuild>false</concurrentBuild>\n  <builders/>\n  <publishers/>\n  <buildWrappers/>\n</project>" }
```

Comment:
also don't need to be global ... only needed in `with auto-config flag` ... if they are used once maybe have them inline in the test